lib/__init__.py

```diff
@@ -0,0 +1,47 @@
+"""Automatically load a configuration according to the RUN_ENV parameter"""
+import os
+import logging
+from importlib.util import spec_from_file_location, module_from_spec
+
+log = logging.getLogger(__name__)
+
+
+def merge_configs(a, b):
+    """Recursively merge config dictionnaries."""
+    new_config = dict()
+    for key in a.keys() ^ b.keys():
+        new_config[key] = a.get(key) or b[key]
+
+    for key in a.keys() & b.keys():
+        if isinstance(a[key], dict) and isinstance(b[key], dict):
+            new_config[key] = merge_configs(a[key], b[key])
+        else:
+            log.warn(
+                'Overriding keys with different types %s and %s' %
+                (type(a[key]), type(b[key])))
+            # TODO: ADD WARNING
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) TODO: ADD WARNING [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=Pylint%3AW0511)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a citation of the person who can best explain this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1707)